README.md

```diff
@@ -4,7 +4,9 @@
 
 Signs and validates HTTP requests.
 
-This projects can help you implements [HMAC](https://en.wikipedia.org/wiki/HMAC) signature to HTTP requests on flutter projects.
+This projects can help you implements [HMAC](https://en.wikipedia.org/wiki/HMAC) signatures to HTTP requests on flutter projects.
+
+For the backend part, use the [RequestsSignature](https://github.com/nventive/RequestsSignature) library to implement [HMAC](https://en.wikipedia.org/wiki/HMAC) signatures.
```

Comment:
Backend can be in many things and not only C#/.NET, should we specify?

Reply:
This recipes currently only exists in .NET and Dart.

Reply:
But someone could use this package for his flutter application and have a rust backend, so he'll have to do the implementation himself? Like we could say that for the .NET BE part use this?

Reply:
yeah we could specify that it's .NET

Reply:
I've reworded to specify backend in .NET